rr_arduino/callbacks.ino

```diff
@@ -17,12 +17,12 @@ void cmdVelocityCallback(const std_msgs::Float32 & cmd_vel_msg)
   velDebugger.publish(&velDebug);
 }
 
-void cmdSteeringCallback(const std_msgs::Float32 & cmd_str_msg)
+void cmd_steering_callback(const std_msgs::Float32 & cmd_str_msg)
```

Comment:
this naming style is not for functions please check this out https://google.github.io/styleguide/cppguide.html#Function_Names